assets/common/src/register.py

```diff
@@ -178,8 +184,7 @@ def main():
                 model_version = str(int(max_version) + 1)
```

Comment:
Replace with `label="latest"` like you were suggesting in the notebook PR?

Reply:
Done